rdap_types/src/lib.rs

```diff
@@ -666,48 +657,20 @@ impl Serialize for NoticeOrRemarkType {
     }
 }
 
-#[skip_serializing_none]
 #[derive(Serialize, Deserialize, Debug)]
 pub struct NoticeOrRemark {
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub title: Option<String>,
-    #[serde(rename = "type")]
-    pub typ: Option<NoticeOrRemarkType>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub r#type: Option<NoticeOrRemarkType>,
+    #[serde(skip_serializing_if = "Option::is_none")]
     pub description: Option<Vec<String>>,
```

Comment:
This is also required based on https://www.rfc-editor.org/rfc/rfc9083.html#section-4.3-2

Reply:
Changed.